nifi-commons/nifi-sensitive-property-provider/src/main/java/org/apache/nifi/properties/SensitivePropertyProvider.java

```diff
@@ -56,4 +56,10 @@
      * @return the raw value to be used by the application
      */
     String unprotect(String protectedValue) throws SensitivePropertyProtectionException;
+
+    /**
+     * Cleans up resources that may have been allocated/used by an SPP implementation
+     * Note: If there is nothing to be done, then this function is a no-op (i.e. no cleanup necessary)
```

Comment:
This particular line depends on the implementation, so it should be removed from the interface documentation.
```suggestion
```

Reply:
Will remove this part of the comment